pkgs/tools/misc/calamares/default.nix

```diff
@@ -15,7 +15,7 @@ mkDerivation rec {
     sha256 = "sha256-TfdLbDsjjPC/8BoEVm4mXePxQ8KX+9jgwKqUR1lcyOk=";
   };
 
-  patches = lib.optionals nixos-extensions [ ./fullnamegs.patch ./nixos-extensions-paths.patch ];
+  patches = lib.optionals nixos-extensions [ ./userjob.patch ./fullnamegs.patch ./nixos-extensions-paths.patch ];
```

Comment:
Would be nice to add some comment explaining each patch, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md

Reply:
I added some basic explanations, but will look over it again tomorrow and maybe improve them a bit